ops/build_and_package.sh

```diff
@@ -34,7 +34,7 @@ go clean
 echo "Building bcda binary..." 
 go build -ldflags "-X main.version=$VERSION"
 echo "Packaging bcda binary into RPM..."
-fpm -v $VERSION -s dir -t rpm -n bcda bcda=/usr/local/bin/bcda swaggerui=/etc/sv/api
+fpm -v $VERSION -s dir -t rpm -n bcda bcda=/usr/local/bin/bcda swaggerui=/etc/sv/api _site==/etc/sv/api
```

Comment:
Should we add a safety check, similar to what we do for Swagger?
https://github.com/CMSgov/bcda-app/pull/162/files#diff-448684739eddd421f889bfa208d8616aR26